packages/typespec-ts/test/integration/generated/mediaTypes/src/clientDefinitions.ts

```diff
@@ -2,28 +2,26 @@
 // Licensed under the MIT license.
 
 import {
-  GetByUnionParameters,
+  GetByUnionOnlyParameters,
```

Comment:
The changes here are because of client.tsp which is expected to ignore it. https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/test/integration/typespec/media-types/client.tsp